package/MDAnalysis/analysis/nucleicacids.py

```diff
@@ -0,0 +1,187 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+"""
+Nucleic acid analysis
+=====================
+
+:Author: Alia Lescoulie
+:Year: 2022
+:copyright: GNU Public Licence v3
+
+"""
+
+from typing import List, Tuple, Dict, Union, NamedTuple
+
+import numpy as np
+import pandas as pd
+from math import pi, sin, cos, atan2, sqrt, pow
+
+import MDAnalysis as mda
+from MDAnalysis.lib import mdamath
+from MDAnalysis.analysis.base import AnalysisBase
+from package import MDAnalysis
+from package.MDAnalysis.exceptions import SelectionError
```

Comment:
There's no package module. This shouldn't even work.... except if you're running from the top of the checked out git repo.

If you want to do development, install with
```
pip install -e package/
```

Reply:
I got the imports fixed not sure why it was like that